framework/db/mssql/QueryBuilder.php

```diff
@@ -35,7 +35,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
         Schema::TYPE_BIGINT => 'bigint',
         Schema::TYPE_FLOAT => 'float',
         Schema::TYPE_DOUBLE => 'float',
-        Schema::TYPE_DECIMAL => 'decimal',
+        Schema::TYPE_DECIMAL => 'decimal(18,0)',
```

Comment:
please use decimal(10,0) for consistency troughout the different DBMSes. See for Mysql https://github.com/yiisoft/yii2/blob/master/framework/db/mysql/QueryBuilder.php#L38